modules/data_release/README.md

```diff
@@ -1,16 +1,40 @@
-# Data Release Module
+# Data Release
 
-The Data Release Module can be used to easily distribute packaged data releases of your study.
+## Purpose
 
-## Features:
+The Data Release module can be used to easily distribute packaged data releases
+of the study.
 
-- Upload new releases through the front-end web interface
-- Differentiate releases using (optionally unique) versioning tag
-- Assign permissions for each release to any specific user
+## Intended users
 
-## Other notes:
+- Researchers and analysis teams
+- Data coordinator that will manage access to, and upload of, the data releases
 
-- Uploads are stored under the modules/data_release/user_uploads directory which can easily be symlinked to another location if necessary, but please create and make sure it is writable by your web server
-- Only superusers have permission to upload files and grant permissions
-- Remove permissions by deleting rows in the data_release_permissions table
-- Upload date will automatically be added during file upload
+## Scope
+
+The module is used to manage, upload and download data releases of the study.
+- Different data releases can be differentiated using the version tag.
+- Data coordinators can upload data releases of the study via the front-end
+web interface.
+- For each data release, data coordinators can manage permissions of any
+  specific user.
+- Upload date will automatically be added during file upload.
+
+
+## Permissions
+
+- Users permissions to view data releases are managed within the data release
+  module.
+- Only superusers have permission to upload and view all data releases.
+  Superusers can also grant other users permissions to specific data release
+  within the data release module.
```

Comment:
This should also go in the hypothetical above ticket.. shouldn't be tied to superuser.

Reply:
agreed. @Jkat could you create a ticket for this please? Thanks!

Reply:
Thanks, added it here: https://redmine.cbrain.mcgill.ca/issues/9599#note-11
There are a few "nice to have" features as well